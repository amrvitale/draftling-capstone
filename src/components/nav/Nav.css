* {
    box-sizing: border-box;
  }
  
  #navbar {
    width: 100%;
    float: left;
    margin: 0;;
    padding: 0;
    list-style-type: none;
    background-color: #f2f2f2;
    border-bottom: 1px solid #ccc; 
    border-top: 1px solid #ccc;
}

#navbar li {
    float: left;
    display: inline;
}

#navbar  a {
    display: block;
    padding: 8px 15px;
    text-decoration: none;
    font-weight: bold;
    color: #069;
    
    text-align: center;
}