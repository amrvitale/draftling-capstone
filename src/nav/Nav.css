* {
    box-sizing: border-box;
  }

  body {
      margin: 0;
  }

  #navbar {
    width: 100%;
    float: left;
    margin: 0;
    padding: 0;
    list-style-type: none;
    background-color: #edf2f4;
    border-bottom: 1px solid #edf2f4;; 
    border-top: 1px solid #edf2f4;;
    text-align: center;
    margin-bottom: 3px;
}

#navbar li {
    float: left;
    display: inline;
    text-align: center;
}

#navbar  a {
    display: inline-block;
    padding: 8px 15px;
    text-decoration: none;
    font-weight: bold;
    color: #D80032;
    text-align: center;
}