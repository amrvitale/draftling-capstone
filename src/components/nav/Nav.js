import React, { Component } from 'react';
import './Nav.css';

class Nav extends React.Component {
    render() {
        return (
            <div>
                <ul id="navbar">
                    <li><a href="#">My Dashboard</a></li>
                    <li><a href="#">Search Other Writer's Draftlings</a></li>
                    <li><a href="#">Logout</a></li>
                </ul>
            </div>

        );
    }
}
export default Nav;