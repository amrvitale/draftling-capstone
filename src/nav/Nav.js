import React, { Component } from 'react';
import './Nav.css';
import { Link } from 'react-router-dom';

class Nav extends React.Component {
    render() {
        return (
            <div>
                <ul id="navbar">
                    <li><Link to='/mydash'>MY DASHBOARD</Link></li>
                    <li><Link to='/about'>ABOUT</Link></li>
                    <li><Link to='/search'>SEARCH DRAFTLINGS</Link></li>
                    <li><a href="#">LOGOUT</a></li>
                </ul>
            </div>

        );
    }
}
export default Nav;