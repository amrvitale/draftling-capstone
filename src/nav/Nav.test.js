import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Nav from './Nav'

describe (`Nav component`, () => {
    it('renders a .Nav by default', () => {
        const wrapper = shallow(<Nav />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})