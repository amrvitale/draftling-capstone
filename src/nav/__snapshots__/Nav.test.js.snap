// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders a .Nav by default 1`] = `
<div>
  <ul
    id="navbar"
  >
    <li>
      <Link
        to="/mydash"
      >
        MY DASHBOARD
      </Link>
    </li>
    <li>
      <Link
        to="/about"
      >
        ABOUT
      </Link>
    </li>
    <li>
      <Link
        to="/search"
      >
        SEARCH DRAFTLINGS
      </Link>
    </li>
  </ul>
</div>
`;
